tests/acceptance/cli/ursula/test_stake_via_allocation_contract.py

```diff
@@ -521,7 +522,8 @@ def test_collect_rewards_integration(click_runner,
                     rest_host='127.0.0.1',
                     rest_port=ursula_port,
                     start_working_now=False,
-                    network_middleware=MockRestMiddleware())
+                    network_middleware=MockRestMiddleware(),
+                    db_filepath=tempfile.mkdtemp())
```

Comment:
Is it possible to use here and in other places some test constant? like `MOCK_URSULA_DB_FILEPATH `. Otherwise I guess many directories will be created when running tests